builder/virtualbox/ovf/builder.go

```diff
@@ -70,6 +174,15 @@ func (b *Builder) Run(ui packer.Ui, hook packer.Hook, cache packer.Cache) (packe
 			GuestAdditionsSHA256: b.config.GuestAdditionsSHA256,
 			Ctx:                  b.config.ctx,
 		},
+		&common.StepDownload{
+			Checksum:     b.config.Checksum,
+			ChecksumType: b.config.ChecksumType,
+			Description:  "OVF/OVA",
```

Comment:
While forcing the extension to be `.ova` on download from a remote url is fine, setting the `Description` to "OVA" alone didn't seem to make sense because this builder is called `virtualbox-ovf` and it works with local OVF files.